src/devices/Tlc1543/Tlc1543.cs

```diff
@@ -0,0 +1,289 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device;
+using System.Device.Gpio;
+using System.Collections.Generic;
+
+namespace Iot.Device.Tlc1543
+{
+    /// <summary>
+    /// Add documentation here
+    /// </summary>
+    public class Tlc1543 : IDisposable
+    {
+        #region Members
+
+        /// <summary>
+        /// Channel used between readings to charge up capacitors in ADC
+        /// </summary>
+        public Channel ChargeChannel = Channel.SelfTest512;
+        private readonly int _chipSelect;
+        private readonly int _dataOut;
+        private readonly int _address;
+        private readonly int _inputOutputClock;
+        private readonly int _endOfConversion;
+        private readonly int _channels = 11;
+        private GpioController _digital = new GpioController(PinNumberingScheme.Logical);
+        private bool _disposedValue;
+
+        #endregion
+
+        #region Constructors
+
+        /// <summary>
+        /// Constructor for Tlc1543
+        /// </summary>
+        /// <param name="ADDR">Address</param>
+        /// <param name="CS">Chip Select</param>
+        /// <param name="DOUT">Data Out</param>
+        /// <param name="IOCLK">I/O Clock</param>
+        public Tlc1543(int ADDR, int CS, int DOUT, int IOCLK)

```

Comment:
You have to have as well a constructor where you pass the GpioController, can be null and use as well the shouldDispose pattern. See here: https://github.com/dotnet/iot/tree/master/src/devices#gpiocontroller

Please use names which are in line with the naming convention, so preffer Address to ADDR, ChipSelect to CS, etc.
Also, fir parameters, the first letter is lower case, so for ADDR, it will be address, for CS chipSelect (note that then you have to keep the uppercase for the next word).
This comment is valid for all the rest of this PR.